docs/src/main/asciidoc/storage.adoc

```diff
@@ -119,6 +119,8 @@ The Spring Boot Starter for Google Cloud Storage provides the following configur
 Base64-encoded contents of OAuth2 account private key for authenticating with the Google Cloud Storage API, if different from the ones in the <<spring-cloud-gcp-core,Spring Framework on Google Cloud Core Module>> | No |
 | `spring.cloud.gcp.storage.credentials.scopes` |
 https://developers.google.com/identity/protocols/googlescopes[OAuth2 scope] for Spring Framework on Google Cloud Storage credentials | No | https://www.googleapis.com/auth/devstorage.read_write
+| `spring.cloud.gcp.storage.universe-domain` | Universe domain of the Storage service. The universe domain is a part of the host that is formatted as `https://${service}.${universeDomain}` | No | Relies on client library’s default universe domain which is `googleapis.com`
+| `spring.cloud.gcp.storage.host` | Host of the Storage service which expects `https://${service}.${universeDomain}` as the format. | No | Relies on client library’s default host which is `https://storage.googleapis.com`
```

Comment:
Do we have any documentation in Storage to describe the relationship between `host` and `universeDomain`? If yes, maybe we can link it here so customers know which one to configure? I know we don't have it yet in pure GAPICs(which is a gap).

Reply:
We don't have any storage-specific documentation on universe domain and host but looks like this is the closest page there is to providing that info? https://cloud.google.com/java/docs/reference/google-cloud-core/latest/com.google.cloud.ServiceOptions.Builder#com_google_cloud_ServiceOptions_Builder_setUniverseDomain_java_lang_String_